ClientApp/viking-vault/src/UserDashboard/ProfilePage.css

```diff
@@ -1,12 +1,8 @@
-.profile-page
-{
-    position: absolute;
-    background-color:white;
-    margin: 15px auto;
-    width: 65%;
-    height: 60%;
-    border-radius : 25px;
-    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+.profile-page {
+    margin: auto;
+    width: 100%;
```

Comment:
You have both width and padding, width alone is 100%, with the padding included the width will be 104%. Is this wanted behavior or and/or you have box-sizing set to border-box?

Reply:
Intended behavior. It is a div in other div and it needs the padding to look good